sigle/src/modules/editor/components/StorySettings.tsx

```diff
@@ -57,10 +50,21 @@ interface Props {
 export const StorySettings = ({ open, onClose, story, onSave }: Props) => {
   return (
     <Dialog open={open} onOpenChange={onClose}>
-      <StyledDialogContent aria-label="Story settings">
+      <StyledDialogContent
+        aria-label="Story settings"
+        css={{
```

Comment:
As this is already a styled content you can move the `css` prop values to `styled(DialogPrimitive.Content, {`